messaging/methods/media/listMedia.md

```diff
@@ -63,6 +63,13 @@ curl -X GET \
 ]
 ```
 
+{% sample lang="java" %}
+
+```java
+ApiResponse<List<Media>> response = controller.listMedia(MESSAGING_ACCOUNT_ID, "");
```

Comment:
Not a review comment but just want to point out that there is an inconsistency across languages with having to set the continuation token as "" or just not having it at all

Reply:
Do you mean null vs ""?

Reply:
More of having the parameter or not. Like in python _I think_ you can do `list_media(account_id)` and completely ignore the parameter

Reply:
Ahhh good point, I can do that in C# because the method is built like:

` public ApiResponse<List<Models.Media>> ListMedia(string userId, string continuationToken = null)`